infra/helper.py

```diff
@@ -264,6 +273,38 @@ def coverage(run_args):
   pipe.communicate()
 
 
+def reproduce(run_args):
+  """Reproduces a testcase in the container."""
+  parser = argparse.ArgumentParser('helper.py reproduce')
+  parser.add_argument('project_name', help='name of the project')
+  parser.add_argument('fuzzer_name', help='name of the fuzzer')
+  parser.add_argument('testcase_path', help='absolute path of local testcase')
```

Comment:
I suggest you translate path to absolute instead. You use `~/Downloads` in examples yourself :)